src/test/javascript/portal/visualise/animations/TemporalExtentSpec.js

```diff
@@ -91,6 +91,26 @@ describe("Portal.visualise.animations.TemporalExtent", function() {
         });
     });
 
+    describe('_createDay', function() {
+        var aMomentInTime = moment.utc('2000-01-01T01:00:00.000');
+        var calledWith = null;
+
+        beforeEach(function() {
+            spyOn(temporalExtent, '_setFirstDay').andCallFake(function(date) { calledWith = date; });
+            spyOn(temporalExtent, '_setLastDay').andCallFake(function(date) { calledWith = date; });
+            temporalExtent._createDay(aMomentInTime);
+        });
+
+        it('calls _setFirstDay', function() {
+            expect(temporalExtent._setFirstDay).toHaveBeenCalled();
+            expect(aMomentInTime.toDate()).toEqual(calledWith.toDate());
+        });
+
+        it('calls _setLastDay', function() {
+            expect(temporalExtent._setLastDay).toHaveBeenCalled();
+            expect(aMomentInTime.toDate()).toEqual(calledWith.toDate());
```

Comment:
Just noting also that this package name is not at all what you'd expect - `Portal.visualise.animations`, I think it should be `Portal.visualise.something` but not sure what the `something` should be.  What do you think?  (btw not expecting you to necessarily do anything about this in this PR).